### ISSUE TITLE
Java examples?

### ISSUE THREAD

**Issue by KoenDG** [bizreach/play-zipkin-tracing#34]
Hello all,

I have no experience in Scala and an existing Play 2.6 project, completely in Java.

Just wondering if examples in Java might be possible? I tried setting it up, but I got no further than logging the url that was called, no further details or anything.

If not, I understand. Time is limited.

**Comment by takezoe** [comment]
I'm not sure about Play with Java, but I guess that an adapter for Java is necessary to do that because play-zipkin-tracing dependes some Scala specific features such as implicit parameter.

**Comment by takezoe** [comment]
Yes. I'm interested in Java support, but I can't start to work on it right now.
